graph/appender/unused_node.go

```diff
@@ -10,11 +10,18 @@ import (
 )
 
 type UnusedNodeAppender struct {
-	GraphType string
+	GraphType   string
+	IsNodeGraph bool
```

Comment:
It is going to be confusing to readers of this what a "Node" graph is. I think you mean "any graph that is not a service-only graph" - but even I'm not sure :) So add a comment here to explain what this boolean is.

Reply:
there was a comment like 3 lines below, but I moved it up here just for you.